test/integration/ipamd/ipamd_suite_test.go

```diff
@@ -67,20 +69,31 @@ var _ = BeforeSuite(func() {
 
 	By("getting node with no pods scheduled to run tests")
 	coreDNSDeployment, err := f.K8sResourceManagers.DeploymentManager().GetDeployment(CoreDNSDeploymentName,
-		CoreDNSDeploymentNameSpace)
+		KubeSystemNamespace)
 	Expect(err).ToNot(HaveOccurred())
 
 	// Copy the deployment to restore later
 	coreDNSDeploymentCopy = coreDNSDeployment.DeepCopy()
 
 	// Add nodeSelector label to coredns deployment so coredns pods are scheduled on 'primary' node
 	coreDNSDeployment.Spec.Template.Spec.NodeSelector = map[string]string{
-		"kubernetes.io/hostname": *primaryInstance.PrivateDnsName,
+		"kubernetes.io/hostname": primaryNode.Name,
 	}
 	err = f.K8sResourceManagers.DeploymentManager().UpdateAndWaitTillDeploymentIsReady(coreDNSDeployment,
 		utils.DefaultDeploymentReadyTimeout)
 	Expect(err).ToNot(HaveOccurred())
 
+	coreDNSAutoscalerDeployment, err := f.K8sResourceManagers.DeploymentManager().GetDeployment(CoreDNSAutoscalerDeploymentName,
```

Comment:
I think we can remove `core-dns-autoscaler` and `ebs-csi-node` daemonset in the automation scripts instead of defining it in our ginkgo tests, since they are not being used anyway

Reply:
Good call, thanks